.circleci/config.yml

```diff
@@ -152,14 +152,38 @@ jobs:
     executor: default-machine
     steps:
       - checkout
-      - restore_cache:
-          key: dependency-cache-{{ .Environment.CIRCLE_SHA1 }}
       - run:
           name: Create dir for test results
           command: mkdir -p ./test/results
       - run:
-          name: Execute integration tests
-          command: npm -s run test:integration
+          name: Set up nvm
+          command: |
+            echo ${NVM_DIR}
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install v12.16.1
```

Comment:
If we are using NVM, which is a great idea, then we should rather just use the [.nvmrc](https://github.com/mojaloop/ml-api-adapter/blob/master/.nvmrc) to determine the node version instead of hard-coding it here.